riscv/csrs.cc

```diff
@@ -654,6 +654,13 @@ bool misa_csr_t::unlogged_write(const reg_t val) noexcept {
       const reg_t new_mevent = state->mevent[i - 3]->read() & ~(MHPMEVENT_VUINH | MHPMEVENT_VSINH);
       state->mevent[i - 3]->write(new_mevent);
     }
+    for (unsigned i = 0; i < proc->TM.count(); i++) {
```

Comment:
Why not `for (trig& : proc->TM.triggers)` and then call `trig->tdata3_read()` directly?

Reply:
The ```proc->TM.triggers``` is private in ```triggers::module_t```. Should we make it public?